lib/jsdom/living/nodes/HTMLInputElement-impl.js

```diff
@@ -252,10 +275,18 @@ class HTMLInputElementImpl extends HTMLElementImpl {
       }
 
       if (candidate !== this) {
-        candidate._checkedness = false;
+        result.push(candidate);
       }
     }
+    return result;
   }
+
+  _removeOtherRadioCheckedness() {
+    this._otherRadioGroupElements.forEach(radioGroupElement => {
+      radioGroupElement._checkedness = false;
+    });
```

Comment:
Style nit: use a for-of loop here instead of forEach.

Reply:
done